.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
https://endoflife.date/nodejs

Node 18 gets you 14 months of support
Node 20 gets you 26

Just sayn

Reply:
Ah, good lookin' out Shea. I'll bring this up with the Squiz folks.

Reply:
Oh yeah I think we need to update to 20 at some point, but currently that's what the squiz devs have been using and it's in the requirements doc, so I didn't want to stir the pot too much 😂 @iamrentman @broleomo perhaps we should bring this up to Squiz at some point?